tests/func/test_add.py

```diff
@@ -475,7 +475,7 @@ def test(self):
         self.assertEqual(0, ret)
 
         foo_stage_file = self.FOO + Stage.STAGE_FILE_SUFFIX
-        # corrupt stage file
+        # corrupt DVC-file
```

Comment:
let's not change stuff that is directly related to code. We are using "srage" files internally, Let's keep it this way. Only change/align with docs the terms that are customer facing.

Reply:
Reverted